newapi/ooniapi/rate_limit_quotas.py

```diff
@@ -19,14 +19,65 @@
 import time
 import ipaddress
 from typing import Dict, List, Optional, Tuple, Union, Set
+from sys import maxsize
+
+import lmdb  # debdeps: python3-lmdb
 
 from ooniapi.config import metrics
 
+LMDB_DIR = "/var/lib/ooniapi/lmdb"
+
 IpAddress = Union[ipaddress.IPv4Address, ipaddress.IPv6Address]
 IpAddrBucket = Dict[IpAddress, float]
 IpAddrBuckets = Tuple[IpAddrBucket, IpAddrBucket, IpAddrBucket]
 TokenBucket = Dict[str, float]
 TokenBuckets = Tuple[TokenBucket, TokenBucket, TokenBucket]
+StrBytes = Union[bytes, str]
+
+
+def ipa_lmdb(ipaddr: IpAddress) -> bytes:
+    return ipaddr.packed
+
+
+class LMDB:
+    def __init__(self, dbnames: tuple):
+        self._env = lmdb.open(LMDB_DIR, metasync=False, max_dbs=10)
```

Comment:
I think having the directory hardcoded here might pose some challenges WRT to running tests in different environments as it requires setup of these dirs.

Reply:
The tests are run in docker at the moment.